packages/client/components/navigation/Dashboard.tsx

```diff
@@ -14,15 +14,19 @@ import {
 
 import NextLink from 'next/link';
 
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-
 import Navbar from './Navbar';
-
-import { faClipboardList, faUsers } from '@fortawesome/free-solid-svg-icons';
+import HireSidebar from './HireSidebar';
+import EarnSidebar from './EarnSidebar';
 
 export default function Dashboard() {
   const sidebar = useDisclosure();
-
+  let isHiring = true;
```

Comment:
Can we navigate to the `earning` section with this? Can you add a temporary button or am I missing something? Looks good in the preview by the way, just would love to see what the `Earning` section looks like

Reply:
This is temporary. I can mock a toggle button tomorrow connecting these conditions. I’ll send a screenshot on discord